### PR TITLE
Refs #16253 - Add max speed var to foreman-proxy-content

### PR DESCRIPTION
**Pull request by sean797**

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -33,6 +33,9 @@
 # $pulp_oauth_secret::                  OAuth secret to be used for Pulp REST interaction
 #                                       type:Optional[String]
 #
+# $pulp_max_speed::                     The maximum download speed for a Pulp task per second. (e.g. "4 Kb" or 1Gb")
+#                                       type:string
```
Comment (stbenjam) [inline manifests/init.pp]: We moved over to Puppet 4 types, can you change this to be `type:Optional[String]`?

**Comment by sean797** [comment]
Not sure why tests are failing, seem unrelated to me?

**Comment by stbenjam** [comment]
It's puppet-lint:

```
%{path}:%{linenumber}:%{KIND}: %{message}
```

There's a bug in the latest version where it's not rendering the messages.  I think it's mostly you have trailing whitespace on some of your lines that needs to be removed.

**Comment by sean797** [comment]
\o/ thanks @stbenjam 👍

**Comment by ekohl** [comment]
Submitted https://github.com/Katello/foreman-installer-modulesync/pull/16 to fix the puppet-lint issue.

**Comment by stbenjam** [comment]
Thanks!

**Pull request merged**